### ISSUE TITLE
Adding numerics to axiom

### ISSUE THREAD

**Issue by lschriml** [OBOFoundry/OBOFoundry.github.io#849]
@pbuttigieg requested discussion of adding numerics to axioms. 

Here's an issue that resurfaces frequently in ENVO and SDGIO 
development, which I think would need an OBO-wide policy:

Many users use classification systems with hard boundaries based on some 
sort of measured value. For example, a climate zone with an upper 
temperature limit of 35*C.

Traditionally, we've shied away from axiomatising these sort of 
thresholds, going instead for more qualitative differentia. However, in 
some cases, the fiat/numeric boundary is really the only difference.

I'm at a stage in ENVO/SDGIO development where I can't really keep 
saying "we don't do that" (important partners like UN Environment would 
find this very odd).

Can we create a policy/guide on how this should be approached? I assume 
the axioms will shortcut a combination of PATO, IAO, UO semantics.

**Comment by cmungall** [comment]
@pbuttigieg I like how you can go to the SIO website and see their main DPs nicely documented with examples, e.g https://github.com/MaastrichtU-IDS/semanticscience/wiki/DP-Measurements

I would like a simple model like this for OBO. 

See also https://github.com/pato-ontology/pato/issues/101

**Comment by jamesaoverton** [comment]
There's a pressing need to document this stuff for OBO. Unless somebody (anybody!) else wants to do it, I will volunteer.

**Comment by cmungall** [comment]
Any progress on this @jamesaoverton (on tomorrow's agenda)

**Comment by pbuttigieg** [comment]
Thanks all. Our use cases would be, e.g., thresholds for canopy density, ice packing, and GDP. 
I realise this drags units into the mix, but if we have a good strategy to coordinate our core this may be tractable.

**Comment by nlharris** [comment]
is this still relevant?

**Comment by matentzn** [comment]
I read this issue 10 times now not sure what it means, but my guess is that this is a request to add a page to OBO somewhere that explains:

1) how to best document data properties
2) how to best document design patterns in an ontologies related to the modelling of quantities

This is probably generally still a good idea.

**Comment by ramonawalls** [comment]
This is still relevant and important for C-Path's work as well as  the quantitative trait work I have been involved with. @jamesaoverton @matentzn @beckyjackson is there bandwidth to make this part of our current contract and maybe include it in OBOok?

**Comment by jamesaoverton** [comment]
@ramonawalls Yes, our plan is to use some of the hours on our contract with you to finish up the quantitative values work that I've shared at COB and OBI workshops, and to integrate that with OBO documentation. I expect to start that in the next few weeks.

**Comment by cmungall** [comment]
On Mon, May 23, 2022 at 10:08 PM Nico Matentzoglu ***@***.***>
wrote:

> I read this issue 10 times now not sure what it means, but my guess is
> that this is a request to add a page to OBO somewhere that explains:
>
>    1. how to best document data properties
>
> See also:
https://github.com/OBOFoundry/COB/issues/195

>
>    1. how to best document design patterns in an ontologies related to
>    the modelling of quantities
>
> See also:
https://github.com/OBOFoundry/COB/issues/35

But Pier's question also involves axiomatizing TBoxes using
quantities/measurements, which is not straightforward. Or rather: it is not
straightforward determining precisely what the actual statement we want to
axiomatize is

See:
https://github.com/oborel/obo-relations/issues/155
